content/en/docs/components/central-dash/profiles.md

```diff
@@ -0,0 +1,238 @@
++++
+title = "Profiles and Namespaces"
+description = "About Kubeflow Profiles and Namespaces for multi-user isolation"
+weight = 20
++++
+
+## What is a Kubeflow Profile?
```

Comment:
It seems to me that this page should be under `multi-tenancy` instead of `Central Dashboard`. How do you think?

Reply:
@zijianjoy I have two strong reasons to keep profiles under the central dashboard section:

1. It's very related to the central dashboard (parts of the guide even have screenshots showing how to manage contributors using the web-ui in the dashboard)
2. We should keep the new "concepts" section about actual concepts, not about how to actually use the components (and I think the profiles page is probably one of the most important pieces of platform-usage docs we have, especially after the update).

Reply:
Although I have preference on the followings, I feel that merging the current PR is going to be helpful, and we can figure out how to adjust in the future:

- Users can manage kubeflow profile via central dashboard right, but they can also manage via yaml. So central dashboard and yaml are only the interfaces of managing multi-tenancy functionality.
- I feel that `concepts` section can document usage guide as well.